### PR TITLE
Added Feature from issue 762

### PR DESCRIPTION
**Pull request by Red-S2**
### Description of the Change

This pr adds the button requested in issue 762 as well as the reminder not do distribute any resources

### Testing

Tested for multiple modloaders and versions

### Related Issues

https://github.com/ATLauncher/ATLauncher/issues/762

### CODE REVIEW

File: src/main/java/com/atlauncher/gui/card/InstanceCard.java [hunk 1]
```diff
@@ -86,6 +86,7 @@ public class InstanceCard extends CollapsiblePanel implements RelocalizationList
     private final JButton serversButton = new JButton(GetText.tr("Servers"));
     private final JButton openWebsite = new JButton(GetText.tr("Open Website"));
     private final JButton openButton = new JButton(GetText.tr("Open Folder"));
+    private final JButton openResourceButton = new JButton(GetText.tr("Open Resources"));
```
Comment (RyanTheAllmighty) [inline src/main/java/com/atlauncher/gui/card/InstanceCard.java:89]: The bottom row of buttons I don't think can show any more at the default size.

Can you copy what was done for the Play buttons where it's a dropdownbutton with the default action being Open Folder and then have this new one in the drop-down like play offline button is

File: src/main/java/com/atlauncher/gui/card/InstanceCard.java [hunk 2]
```diff
@@ -502,6 +504,12 @@ private void addActionListeners() {
                         Constants.SERVERS_LIST_PACK, instance.getSafePackName())));
         this.openWebsite.addActionListener(e -> OS.openWebBrowser(instance.getWebsiteUrl()));
         this.openButton.addActionListener(e -> OS.openFileExplorer(instance.getRoot()));
+        this.openResourceButton.addActionListener(e -> {
+            DialogManager.okDialog().setTitle("Reminder")
```
Comment (RyanTheAllmighty) [inline src/main/java/com/atlauncher/gui/card/InstanceCard.java:508]: Any text should be wrapped in `GetText.tr()`

File: src/main/java/com/atlauncher/gui/card/InstanceCard.java [hunk 3]
```diff
@@ -502,6 +504,12 @@ private void addActionListeners() {
...
+                    .setContent("You may not distribute ANY resources.")
```
Comment (RyanTheAllmighty) [inline src/main/java/com/atlauncher/gui/card/InstanceCard.java:509]: Same here for `GetText.tr()`

File: src/main/java/com/atlauncher/gui/card/InstanceCard.java [hunk 4]
```diff
@@ -502,6 +504,12 @@ private void addActionListeners() {
...
+                    .setType(DialogManager.OK_OPTION).show();
```
Comment (RyanTheAllmighty) [inline src/main/java/com/atlauncher/gui/card/InstanceCard.java:510]: This should probably be the `WARNING` constant from DialogManager

**Review by RyanTheAllmighty — CHANGES_REQUESTED** [review]

**Comment by Red-S2** [comment]
pls review

**Review by RyanTheAllmighty — CHANGES_REQUESTED** [review]

File: src/main/java/com/atlauncher/gui/card/InstanceCard.java [hunk 5]
```diff
@@ -85,9 +85,19 @@ public class InstanceCard extends CollapsiblePanel implements RelocalizationList
     private final JButton editButton = new JButton(GetText.tr("Edit Mods"));
     private final JButton serversButton = new JButton(GetText.tr("Servers"));
     private final JButton openWebsite = new JButton(GetText.tr("Open Website"));
-    private final JButton openButton = new JButton(GetText.tr("Open Folder"));
     private final JButton settingsButton = new JButton(GetText.tr("Settings"));
 
+    private final JPopupMenu openPopupMenu = new JPopupMenu();
+    private final JMenuItem openFolderMenuItem = new JMenuItem(GetText.tr("Open Folder"));
+    private final JMenuItem openResourceMenuItem = new JMenuItem(GetText.tr("Open Resources"));
+    private final DropDownButton openButton = new DropDownButton(GetText.tr("Open Files"), openPopupMenu, true,
```
[the remark this answers is not in the document]
Comment (Red-S2) [reply src/main/java/com/atlauncher/gui/card/InstanceCard.java]: Done

**Pull request merged**